_data/packages.yml

```diff
@@ -513,7 +550,7 @@
 - package_name: rdata
   package_description: Read R datasets from Python.
   submitting_author:
-      name: Carlos Ramos Carreño
+      name: Name
```

Comment:
This is the weirdest thing! this person has a name listed on github so i have no idea where Name is coming from here. this is a blatant bug. but it's a new bug that didn't used to exist.

Reply:
again this is in our template. i cleaned it up! resolving this comment as it will be fixed when we rerun the bot